tests/post_training/model_scope.py

```diff
@@ -75,6 +75,17 @@
         "backends": [BackendType.FX_TORCH, BackendType.TORCH, BackendType.CUDA_TORCH, BackendType.OV, BackendType.ONNX],
         "batch_size": 128,
     },
+    {
+        "reported_name": "torchvision/mobilenet_v3_small",
+        "model_id": "mobilenet_v3_small",
+        "pipeline_cls": ImageClassificationTorchvision,
+        "compression_params": {
+            "fast_bias_correction": False,
+            "preset": QuantizationPreset.MIXED,
+        },
+        "backends": [BackendType.FX_TORCH, BackendType.OV, BackendType.ONNX],
```

Comment:
Why only these backends? Is this about BiasCorrection support? If so, add the `BC` suffix for the `model_id`.